README.md

```diff
@@ -1,114 +1,112 @@
 
-- **[Krypton](https://modrinth.com/mod/krypton)** (Slightly reduces memory usage on servers, decreases server CPU usage and server ticks)
+- **[Krypton](https://modrinth.com/mod/krypton)** (Slightly reduces memory usage on servers, decreases server CPU usage and server ticks. Not in 1.20.4)
 
-- **[LazyDFU](https://modrinth.com/mod/lazydfu)** (Improves game boot time)
+- **[LazyDFU](https://modrinth.com/mod/lazydfu)** (Improves game boot time.)
 
-- **[Lithium](https://modrinth.com/mod/lithium)** (General purpose performance mod)
+- **[Lithium](https://modrinth.com/mod/lithium)** (General purpose performance mod. Not included in 1.20.4)
 
-- **[More Culling](https://modrinth.com/mod/moreculling)** (Culling mod to improve performance) 
+- **[ModernFix](https://modrinth.com/mod/modernfix)** (Performance mod, reduces memory usage yet again.)
 
-- **[Noxesium](https://modrinth.com/mod/noxesium)** (Adds features to fix bugs and performance)
+- **[Noxesium](https://modrinth.com/mod/noxesium)** (Adds features to fix bugs and performance. Only in 1.19.2/4)
 
-- **[Smooth Boot](https://modrinth.com/mod/smoothboot-fabric)** (Makes Minecraft loading smoother)
+- **[ServerCore](https://modrinth.com/mod/servercore)** (Reduces lag spikes in both single-player and servers. Not included in 1.18.2/1.19.2) 
 
-- **[Sodium](https://modrinth.com/mod/sodium)** (Improves FPS and fixes graphical issues)
+- **[Smooth Boot](https://modrinth.com/mod/smoothboot-fabric)** (Makes Minecraft loading smoother.)
 
-- **[Starlight](https://modrinth.com/mod/starlight)** (Improves the lighting engine)
+- **[Sodium](https://modrinth.com/mod/sodium)** (Improves FPS and fixes graphical issues.)
 
-**Iris will only work on 1.18.2 and maybe even 1.19.2; please remember that FPS will not be high, and crashing is more likely!** <br>
-**There may be more mods, please check out [this](https://github.com/QuestCraftPlusPlus/Pojlib/blob/QuestCraft/mods.json) for aditional information.**
+- **[Starlight](https://modrinth.com/mod/starlight)** (Improves the lighting engine. Not included in 1.20.1 and later)
 
-## Some Recommended Settings and Tips (For Zink)
-1. In the options menu, go to VR Settings/Stereo Rendering and set the resolution (NOT CAMERA RES) to 80%. This will decrease the general quality but will give an extra performance boost.
-2. The game's render distance is best at 4-6, but you *may* encounter lag spikes when using it. 9 and above is not recommended.
-3. Oceans contain LOTS of kelp that decreases framerate. Don't mine any of it all at once, as your game may CRASH if many entities (in this case, kelp waiting to be picked up) exist simultaneously.
-4. **[Skyblock](https://minecraft.wiki/w/Tutorials/Skyblock)** and **[Oneblock](https://www.curseforge.com/minecraft/worlds/oneblock)** worlds perform the best, as they require less hardware usage.
+**A table showing these mods and their supported versions in a better and more visually appealing way is coming soon!**\
```

Comment:
it is?

Reply:
Yeah I'm gonna work on one when I have the time to. Which is it, a very long list or a cool table?

Reply:
I'm not gonna include this text in the final version of this PR, but my next PR will include this table.